R/read_csv.R

```diff
@@ -121,6 +121,8 @@ read_cmdstan_csv <- function(files,
   col_types <- NULL
   col_select <- NULL
   not_matching <- c()
+  vroom_warnings <- 0
```

Comment:
Keep track of whether we've thrown a warning so that we can avoid throwing it for every csv file. Once per call to read_cmdstan_csv() seems like enough.